src/config/baselines.tsx

```diff
@@ -0,0 +1,55 @@
+import nsoDisabled from '../data/nso/NSO_Disabled_Admin1_Total.json';
```

Comment:
Should we put this in a subdirectory of `config`? I think the original intention was for everything that might change country-to-country in `config`, so that you could be confident that that's the only place to make changes... but we could just add this `data` directory too.

Reply:
Hmm good point, not urgent but let's think about how we want to handle country specific data